fintorch/models/timeseries/tft/tft_module.py

```
@@ -63,18 +99,78 @@
-        static_inputs: Dict[str, int],
+        # New granular feature parameters
+        num_past_target_features: int,
+        num_past_known_cov_features: int,
+        num_past_unknown_cov_features: int,
+        num_future_known_cov_features: int,
+        num_static_real_features: int,
+        num_static_categorical_features: int,
+        static_categorical_cardinalities: List[int],
         batch_size: int,
         device: str,
         quantiles: List[float] = [0.1, 0.5, 0.9],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)